msg/HumanStateMsg.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+geometry_msgs/Point position
```

Comment:
I think we should make this message type to be Pose instead of Point, so that we can consider orientation. For now we can just fill in default orientation, but I do think we need Pose instead of Point

Reply:
Mentioned below as well. We want to have the message as an array of human. Probably create another message type HumanStateArrayMsg, and make this HumanStateMsg as the type inside that array type. You may want to also remove header here and put header in that array message.